internal/pkg/cli/env_list.go

```diff
@@ -133,22 +119,25 @@ func (o *listEnvOpts) jsonOutput(envs []*config.Environment) (string, error) {
 	return fmt.Sprintf("%s\n", b), nil
 }
 
-// BuildEnvListCmd builds the command for listing environments in a project.
+// BuildEnvListCmd builds the command for listing environments in an application.
 func BuildEnvListCmd() *cobra.Command {
 	vars := listEnvVars{
 		GlobalOpts: NewGlobalOpts(),
 	}
 	cmd := &cobra.Command{
 		Use:   "ls",
-		Short: "Lists all the environments in a project",
+		Short: "Lists all the environments in an application",
```

Comment:
nit: can we end with a period "." for consistency

Reply:
done!